### PR TITLE
chore(demo): markup section as standalone

### PR DESCRIPTION
**Pull request by splincode**

### CODE REVIEW

**Review by waterplea — APPROVED** [review]

File: projects/demo/src/modules/components/island/examples/1/index.ts [hunk 1]
```diff
@@ -1,12 +1,15 @@
 import {Component} from '@angular/core';
 import {changeDetection} from '@demo/emulate/change-detection';
 import {encapsulation} from '@demo/emulate/encapsulation';
+import {TuiLinkDirective} from '@taiga-ui/core';
+import {TuiIslandModule} from '@taiga-ui/kit';
 
 @Component({
-    selector: 'tui-island-example-1',
+    standalone: true,
+    imports: [TuiIslandModule, TuiLinkDirective],
```
Comment (nsbarsukov) [inline projects/demo/src/modules/components/island/examples/1/index.ts:9]: image



https://taiga-ui--pr7497-splincode-markup-rr0ywop3.web.app/components/island#base

File: projects/demo/src/modules/markup/breakpoints/index.html [hunk 2]
```diff
@@ -49,10 +49,9 @@
         <tui-doc-example
             id="usage"
             heading="Usage"
-            [content]="example1"
-        >
-            <example-css-breakpoints-1 />
-        </tui-doc-example>
+            [component]="1 | tuiComponent"
+            [content]="1 | tuiExample"
```
Comment (nsbarsukov) [inline projects/demo/src/modules/markup/breakpoints/index.html:53]: Снимок экрана 2024-05-21 в 10 41 47

https://taiga-ui--pr7497-splincode-markup-rr0ywop3.web.app/breakpoints

File: projects/demo/src/modules/markup/colors/examples/table/table.style.less [hunk 3]
```diff
@@ -1,4 +1,4 @@
-@import '@taiga-ui/core/styles/taiga-ui-local';
+@import '../../../../../../../../node_modules/@taiga-ui/core/styles/taiga-ui-local';
```
Comment (nsbarsukov) [inline projects/demo/src/modules/markup/colors/examples/table/table.style.less:1]: 🤔

Comment (nsbarsukov) [inline projects/demo/src/modules/markup/colors/examples/table/table.style.less:1]: 🤔

File: projects/demo/src/modules/markup/form/index.html [hunk 4]
```diff
@@ -88,9 +88,8 @@
         <tui-doc-example
             id="form"
             heading="Basic"
-            [content]="example1"
-        >
-            <tui-form-example-1 />
-        </tui-doc-example>
+            [component]="1 | tuiComponent"
+            [content]="1 | tuiExample"
```
Comment (nsbarsukov) [inline projects/demo/src/modules/markup/form/index.html:92]: Снимок экрана 2024-05-21 в 10 44 24

https://taiga-ui--pr7497-splincode-markup-rr0ywop3.web.app/form

File: projects/demo/src/modules/markup/spaces/index.html [hunk 5]
```diff
@@ -32,17 +32,15 @@
         <tui-doc-example
             id="classes"
             heading="Classes"
-            [content]="example1"
-        >
-            <tui-spaces-example-1 />
-        </tui-doc-example>
+            [component]="1 | tuiComponent"
+            [content]="1 | tuiExample"
```
Comment (nsbarsukov) [inline projects/demo/src/modules/markup/spaces/index.html:36]: Снимок экрана 2024-05-21 в 10 47 12

https://taiga-ui--pr7497-splincode-markup-rr0ywop3.web.app/spaces

File: projects/demo/src/modules/markup/tables/index.ts
Comment (nsbarsukov) [inline projects/demo/src/modules/markup/tables/index.ts:1]: Run demo application locally and double check ALL changed pages, please

**Review by nsbarsukov — CHANGES_REQUESTED** [review]

File: projects/demo/src/modules/markup/tables/index.ts
Comment (nsbarsukov) [reply projects/demo/src/modules/markup/tables/index.ts:1]: Снимок экрана 2024-05-21 в 10 49 03

https://taiga-ui--pr7497-splincode-markup-rr0ywop3.web.app/tables

File: projects/demo/src/modules/components/island/examples/1/index.ts (hunk 1 above)
Comment (splincode) [reply projects/demo/src/modules/components/island/examples/1/index.ts]: fixed

File: projects/demo/src/modules/markup/breakpoints/index.html (hunk 2 above)
Comment (splincode) [reply projects/demo/src/modules/markup/breakpoints/index.html:54]: fixed

File: projects/demo/src/modules/markup/colors/examples/table/table.style.less (hunk 3 above)
Comment (splincode) [reply projects/demo/src/modules/markup/colors/examples/table/table.style.less]: fixed

File: projects/demo/src/modules/markup/lists/index.html [hunk 6]
```diff
@@ -4,63 +4,49 @@
     <tui-doc-example
         id="large"
         heading="large"
-        [content]="example1"
+        [content]="1 | tuiExample: 'ts'"
```
[the remark this answers is not in the document]
Comment (splincode) [reply projects/demo/src/modules/markup/lists/index.html:11]: fixed

File: projects/demo/src/modules/markup/spaces/index.html (hunk 5 above)
Comment (splincode) [reply projects/demo/src/modules/markup/spaces/index.html:39]: fixed

File: projects/demo/src/modules/markup/tables/index.ts
Comment (splincode) [reply projects/demo/src/modules/markup/tables/index.ts:1]: fixed

File: projects/demo/src/modules/markup/form/examples/1/index.less [hunk 7]
```diff
@@ -1,4 +1,4 @@
-@import '@taiga-ui/core/styles/taiga-ui-local';
+@import '../../../../../../../../node_modules/@taiga-ui/core/styles/taiga-ui-local';
```
Comment (nsbarsukov) [inline projects/demo/src/modules/markup/form/examples/1/index.less:1]: Should we revert this line ?

**Review by nsbarsukov — APPROVED** [review]

**Pull request merged**